pandas/core/frame.py

```diff
@@ -949,16 +954,21 @@ def to_dict(self, orient='dict', into=dict):
             instance of the mapping type you want.  If you want a
             collections.defaultdict, you must pass it initialized.
 
-            .. versionadded:: 0.21.0
+            .. versionadded:: 0.21.0.
 
         Returns
         -------
         result : collections.Mapping like {column -> {index -> value}}
 
+        See Also
+        --------
+        DataFrame.from_dict: create a DataFrame from a dictionary
+
```

Comment:
I'm guessing if it'd make sense to also add `to_json` here. I think users checking this method can be looking for that.